index.js

```diff
@@ -1,4 +1,5 @@
 module.exports = {
   ui: require('./ui'),
-  api: require('./api')
+  api: require('./api'),
+  S3: require('./clients/s3')
```

Comment:
It might be good if there was an index file in `./clients` also, so we can `const { S3 } = require('@asl/service/clients');`

`require`ing the root is going to involve pulling _a lot_ of extra code into memory to just get the S3 client.

Reply:
done